tests/scripts/run_performance.sh

```diff
@@ -84,17 +88,21 @@ run_device_perf_models() {
     #Model Device perf regression tests to make sure thy run on no-soft-reset BMs
     tests/scripts/run_profiler_regressions.sh PROFILER
```

Comment:
This should be under GS as well. WH does this test as part of post commit already.